experiments/tester/src/RNTester/TestComponents/ThemeTest.tsx

```diff
@@ -138,22 +157,39 @@ const ThemeTestInner: React.FunctionComponent = () => {
   const themedStyles = getThemedStyles(useTheme());
   const onAppChange = React.useCallback((app: string) => {
     brand = app;
-    customRegistry.setTheme(fakeBrandTheme, 'BrandedTheme');
+    // Invalidate the DAG children of the shimmed brand colors
+    customRegistry.setTheme(fakeBrandTheme, 'Default');
+    customRegistry.setTheme(fakeBrandTheme, 'WhiteColors', 'RealWhiteColors');
   }, []);
+
+  const [theme, setTheme] = React.useState('Default');
+  const onThemeChange = React.useCallback(
+    (theme: string) => {
+      setTheme(theme);
+    },
+    [theme, setTheme]
+  );
   return (
     <ReactNative.View>
-      <RadioGroup label="Pick App Colors" onChange={onAppChange} defaultSelectedKey="Office">
-        <RadioButton buttonKey="Office" content="Office" />
-        {Object.keys(brandColors).map((app: string) => (
-          <RadioButton key={app} buttonKey={app} content={app} />
-        ))}
-      </RadioGroup>
-      <Text style={themedStyles.extraLargeStandardEmphasis}>Platform Theme</Text>
+      <Text style={themedStyles.extraLargeStandardEmphasis}>Configure Theme</Text>
       <Separator />
-      <Panel />
-      <Text style={themedStyles.extraLargeStandardEmphasis}>Custom JS Theme</Text>
+      <ReactNative.View style={styles.pickerContainer}>
+        <RadioGroup label="Pick App Colors" onChange={onAppChange} defaultSelectedKey="Office">
```

Comment:
As mentioned above, I think `<Picker>` is a better choice.

Reply:
Yea, but I'm not gonna change it now, sorry.  :)